src/jit/valuenum.h

```diff
@@ -341,46 +343,79 @@ class ValueNumStore
     // It returns NoVN for a "typ" that has no one value, such as TYP_REF.
     ValueNum VNOneForType(var_types typ);
 
-    // Return the value number representing the singleton exception set containing the exception value "x".
+    // Returns the value number for negative one of the given "typ".
+    // It returns NoVN for a "typ" that has no negative one value, such as TYP_REF, or TYP_UINT
+    ValueNum VNNegOneForType(var_types typ);
```

Comment:
Is this used? Can't seem to find its definition nor any uses in the diff. I was wondering if it perhaps assert for unsigned types.

Reply:
I think it will be used with Integer Division (MAX_INT / -1) throws an overflow exception